eclipse-theia/theia
# Fix for WebviewPanelSerializer not working in Theia

#### What it does
Fixes #12834

For a detailed root cause analysis, refer to this [comment](https://github.com/eclipse-theia/theia/issues/12834#issuecomment-1685357196)

#### How to test

1. Clone the test repository from issue #12834 
2. cd into the "myext" folder and create a packaged vscode plugin by running the "vsce package" command
3. Copy the packaged vsix file into this issue's git repository and paste it into the "plugins" folder
4. Run the application and see if you're able to reproduce issue #12834 

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
